dev.atoft.Reactions.json

```diff
@@ -0,0 +1,107 @@
+{
+    "app-id" : "dev.atoft.Reactions",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.ziglang"],
+    "command" : "/app/bin/reactions",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--env=GTK_MEDIA=ffmpeg"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/ziglang"
+    },
+    "cleanup" : [
+        "/bin/gtk4-*",
+        "/include",
+        "/lib/debug",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/applications/org.gtk*",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/metainfo",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a",
+        "/zig"
+    ],
+    "modules" : [
+        {
+            "name" : "libsass",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lazka/libsass.git",
+                    "branch" : "meson"
+                }
+            ]
+        },
+        {
+            "name" : "sassc",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lazka/sassc.git",
+                    "branch" : "meson"
+                }
+            ]
+        },
+        {
+            "name" : "gtk",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dmedia-ffmpeg=enabled"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/gtk.git",
+                    "tag" : "4.4.0",
+                    "commit" : "f1f197e3b94a55d5cbfaae2498f991a0ae733b32"
```

Comment:
the gtk version in the runtime isn't enough?

Reply:
I needed to use the ffmpeg backend due to https://gitlab.gnome.org/GNOME/gtk/-/issues/4062. I don't think it's available in any runtime?